src/components/layout/Accordion/AccordionItem.tsx

```diff
@@ -0,0 +1,184 @@
+								return (
+									<TagSummary
+										className={classnames(
+											styles.AccordionItem_Summary,
+											'AccordionItem_Summary',
+											{
+												[styles.AccordionItem_Summary__ClickAreaAll]: clickArea === 'all',
+												['AccordionItem_Summary__ClickAreaAll']: clickArea === 'all',
+											}
+										)}
+										onClick={onClickSummary}
+									>
+										{React.cloneElement(child as React.ReactElement, {})}
+										{/* don't render if icon is disabled otherwise this can be tabbed to */}
+										{icon !== false && (
+											<button
+												className={classnames(
+													styles.AccordionItem_Summary_Action,
+													'AccordionItem_Summary_Action',
+													{
+														// allow developers to override for custom open/close styles
+														['AccordionItem_Summary_Action__Opened']: opened,
+													}
+												)}
+												onClick={onClick}
+												type="button"
+											>
+												{!opened && icon === true && (
+													<CaretIcon />
+												)}
+												{opened && icon === true && (
+													<CheckmarkMixedIcon width={14} height={3} />
+												)}
+												{typeof icon === "object" && (
+													<>
+														{icon}
+													</>
+												)}
+											</button>
+										)}
+									</TagSummary>
+								);
+							case 1:
+								return (
+									<AnimateHeight
+										animationStateClasses={animationStateClasses}
+										className={classnames(
+											'AccordionItem_Content',
+										)}
+										duration={ 200 }
+										height={ opened ? 'auto' : 0 }
+									>
+										{child}
+									</AnimateHeight>
+								);
+							default:
+								return React.cloneElement(child as React.ReactElement, {});
+						}
+					})}
+				</>
+			</div>
+		</Container>
+	);
+};
+
+interface PropsUncontrolled extends PropsBase {
+	openedDefault?: boolean,
+}
+
+const AccordionItemUncontrolled: React.FC<PropsUncontrolled> = ({
+	openedDefault,
+	onToggle,
+	...otherProps
+}) => {
+	const [isOpen, setIsOpen] = React.useState(openedDefault ?? false);
+
+	const onClick = (itemId: PropsBase['itemId']) => {
+		// toggle open state
+		setIsOpen((isOpen) => !isOpen);
+
+		// call optional callback
+		onToggle?.(itemId);
+	};
+
+	return <AccordionItemControlled opened={isOpen} onToggle={onClick} {...otherProps} />;
+};
+
+function instanceOfPropsControlled(object: any): object is PropsControlled {
+    return 'opened' in object;
+}
+
+/**
+ * This is a factory of sorts that determines whether to use the controlled or uncontrolled variation.
+ * Uses controlled-specific prop(s) to determine whether the parent is trying to "control" state or not.
+ */
+const AccordionItem : React.FC<PropsControlled | PropsUncontrolled> = (props) => {
+	return instanceOfPropsControlled(props) ? (
+		<AccordionItemControlled
+			{...props}
+		/>
+	) : (
+		<AccordionItemUncontrolled
+			{...(props)}
```

Comment:
```suggestion
			{...props}
```

Reply:
Oh ha! I'm not even sure how this happened 😅 Consider it changed.